Controller/TblCronTaskController.php

```diff
@@ -0,0 +1,234 @@
+        ));
+    }
+
+    /**
+     * Finds and displays a TblCronTask entity.
+     *
+     * @param int $id
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function showAction($id)
+    {
+        $cron = $this->get('trovit.cron_manager.read_cron_task')->getCronById($id);
+
+        if (!$cron) {
+            throw $this->createNotFoundException('Unable to find TblCronTask entity.');
+        }
+
+        $deleteForm = $this->createDeleteForm($id);
+
+        return $this->render('TrovitCronManagerBundle:TblCronTask:show.html.twig', array(
+            'cron'        => $cron,
+            'delete_form' => $deleteForm->createView(),
+        ));
+    }
+
+    /**
+     * Displays a form to edit an existing TblCronTask entity.
+     *
+     * @param int $id
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function editAction($id)
+    {
+        $cron = $this->get('trovit.cron_manager.read_cron_task')->getCronById($id);
+
+        if (!$cron) {
+            throw $this->createNotFoundException('Unable to find TblCronTask entity.');
+        }
+
+        $editForm = $this->createEditForm($cron);
+        $deleteForm = $this->createDeleteForm($id);
+
+        return $this->render('TrovitCronManagerBundle:TblCronTask:edit.html.twig', array(
+            'cron'        => $cron,
+            'edit_form'   => $editForm->createView(),
+            'delete_form' => $deleteForm->createView(),
+        ));
+    }
+
+    /**
+    * Creates a form to edit a TblCronTask entity.
+    *
+    * @param TblCronTask $entity The entity
+    * @return \Symfony\Component\Form\Form The form
+    */
+    private function createEditForm(TblCronTask $entity)
+    {
+        $form = $this->createForm(new TblCronTaskType(), $entity, array(
+            'action' => $this->generateUrl('tblcrontask_update', array('id' => $entity->getId())),
+            'method' => 'PUT',
+        ));
+
+        $form->add('submit', 'submit', array('label' => 'Update'));
+
+        return $form;
+    }
+
+    /**
+     * Edits an existing TblCronTask entity.
+     *
+     * @param Request $request
+     * @param int $id
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+     * @throws CommandNotExistsException
+     */
+    public function updateAction(Request $request, $id)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $cron = $em->getRepository('TrovitCronManagerBundle:TblCronTask')->find($id);
+
+        if (!$cron) {
+            throw $this->createNotFoundException('Unable to find TblCronTask entity.');
+        }
+
+        $deleteForm = $this->createDeleteForm($id);
+        $editForm = $this->createEditForm($cron);
+        $editForm->handleRequest($request);
+
+        if ($editForm->isValid()) {
+            if (!$this->get('trovit.cron_manager.command_validator')->commandExists($cron->getCommand())) {
```

Comment:
To unify all create operations, use persistCron instead of commandExists and then flush.